src/config.rs

```diff
@@ -2467,6 +2467,35 @@ impl LogConfig {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, PartialEq, Debug)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct CausalTsConfig {
```

Comment:
It should be put into resolve-ts module.

Reply:
Could you give more comments about why ?
`causal_ts` has quite different functions with `resolved-ts`. Please also refer to [this discussion](https://github.com/tikv/tikv/pull/12099#discussion_r823352929) before.

Reply:
I don't see there is any discussion but @overvenus is making a similar suggestion. `CausalTsConfig` works for causal timestamp, and causal timestamp is defined in its own module, so it's intuitive to put them together.

Reply:
En... You mean put into `causal-ts` module ? OK, I see.

Reply:
Sorry, I just found I wrote the wrong ts 🤣

Reply:
Never mind ~